charts/loki/values.yaml

```diff
@@ -49,6 +49,13 @@ tracing:
 config:
   # existingSecret:
   auth_enabled: false
+
+  memberlist:
+    bind_port: 7946
```

Comment:
I think this is the default, so we shouldn't need it

Reply:
removed. thanks